src/Carbon/CarbonInterval.php

```diff
@@ -621,6 +621,30 @@ public function add(DateInterval $interval)
         return $this;
     }
 
+    /**
+     * Multiply current instance given number of times
+     *
+     * @param float $factor
+     *
+     * @return static
+     */
+    public function times($factor)
+    {
+        if ($factor < 0) {
+            $this->invert = !$this->invert;
```

Comment:
`$this->invert = 1 - $this->invert;`

Reply:
These two will be equivalent due to type casting. Is there some other reason than cosmetics? If no then what about stylish `1 ^ $this->invert`? ^^

Reply:
Is there a reason this should be an integer instead of a boolean?

Reply:
@36864 Yes, DateInterval compatibility.
@imbrish Bitwise operator is OK.